opencodelists/forms.py

```diff
@@ -49,6 +49,7 @@ class CodelistCreateForm(forms.Form):
         for id, system in CODING_SYSTEMS.items()
         if issubclass(system, BuilderCompatibleCodingSystem)
     )
+    CODING_SYSTEM_CHOICES.sort(key=lambda x: x[1].lower())
```

Comment:
This would probably be better implemented as a change to the previous statement rather than re-sorting an already-sorted list. There's no need for the intermediate sorting

Reply:
```py
    CODING_SYSTEM_CHOICES = [("", "---")] + sorted(
        [
            (id, system.name)
            for id, system in CODING_SYSTEMS.items()
            if issubclass(system, BuilderCompatibleCodingSystem)
        ],
        key=lambda x: x[1].lower(),
    )
```

Is this what you mean?

Reply:
yep :)